File: cqrs-es/az-func-ts/src/functions/commands/UpdateContactInfoCommand.ts
export interface EmailUpdateRequest {
    email: string;
    type: string;
}

export interface PhoneUpdateRequest {
    phoneNumber: string;
    type: string;
}

export interface UpdateContactInfoCommand {
    partyId: string;
    emails: EmailUpdateRequest[];
    phoneNumbers: PhoneUpdateRequest[];
}

File: cqrs-es/az-func-go/UpdateContactInfo/commands/update_contact_info_command.go
package commands

import (
	"az-func-go/UpdateContactInfo/model"
)

// UpdateContactInfoCommand represents the command to update contact info

type UpdateContactInfoCommand struct {
	PartyId      string                  `json:"partyId"`
	Emails       []model.UpdateEmailItem `json:"emails"`
	PhoneNumbers []model.UpdatePhoneItem `json:"phoneNumbers"`
}
